packages/commonwealth/server/workers/evmChainEvents/nodeProcessing.ts

```diff
@@ -72,6 +97,60 @@ export async function processChainNode(
         await lastProcessedBlock.save({ transaction });
       }
 
+      const contestEvents = events
+        .map((event) => {
+          const timestamp = new Date(); // TODO: get block timestamp
```

Comment:
If `created_at` is not one of the emitted event properties it should not be part of the `event_payload` and it should be fetched in the event handler. Chain-events should not augment data. Especially when it's not clear that the `created_at` property will actually be the date that the event was processed rather than when the event was made a part of the blockchain.

Reply:
Good point. I removed the property.